src/main/java/com/redhat/devtools/lsp4ij/LanguageServerManager.java

```diff
@@ -0,0 +1,57 @@
+package com.redhat.devtools.lsp4ij;
+
+import com.intellij.openapi.project.Project;
+import org.jetbrains.annotations.NotNull;
+
+public class LanguageServerManager {
+
+    private final Project project;
+
+
+    public static LanguageServerManager getInstance(@NotNull Project project) {
+        return project.getService(LanguageServerManager.class);
+    }
+
+    private LanguageServerManager(Project project) {
+        this.project = project;
+    }
+
+    public void stop(String languageSeverId) {
+        var serverDefinition = LanguageServersRegistry.getInstance().getServerDefinition(languageSeverId);
+        if (serverDefinition == null) {
+            return;
+        }
+        for (var ls : LanguageServiceAccessor.getInstance(project).getStartedServers()) {
+            if (serverDefinition == ls.getServerDefinition()) {
+                ls.stop();
+            }
+        }
+    }
+
+    public void start(String languageSeverId) {

```

Comment:
Consider adding `start(String languageSeverId, StartOptions options) {}` where StartOptions can have a `boolean forceStart` property (false by default), for starting LS without opened files. Making it an class allows us to add more options in the future without changing the API

Reply:
fixed and forceStart  is implemnted and it the default behavior.